fds/services/dvc_service.py

```diff
@@ -128,6 +125,10 @@ def __get_to_add_to_dvc(self,
                 if os.path.isdir(file_or_dir_to_check):
                     return None, file_or_dir_to_check
                 return None, None
+            # If the file or dir is greater than threshold and is dvc ignored
+            dvc_output = check_dvc_ignore(file_or_dir_to_check)
+            if convert_bytes_to_string(dvc_output.stdout) != '':
+                return None, file_or_dir_to_check
```

Comment:
This is preserving the same logic that existed before:
```suggestion
                return None, None
```

Reply:
First element of the tuple is folder/file to be added
Second element of tuple is folder/file to be ignored. So we would like to ignore the folder (so we don't scan any of the files inside the folder) and ignore file if its a file. 

If you return None in second element of tuple we don't ignore it and we will keep tracking the files inside it (if the folder was dvc ignored)

Reply:
This is a change of behavior from what was before, and also what happens if a file is ignored by dvc but not git?
We don't want to skip it in that case, we want to `git add` it - and the other way around as well, I may want to dvc add a gitignored file. 
We may need to remove impossible options from the prompt in those cases, and only skip the file completely if it's ignored by both

Reply:
I don't think this is change of behavior from what we have currently. This file change is only for DVC, if the files are ignored here they are added to git (unless they are .gitignored). 

I think currently if a file/folder is `.gitignored` we don't show them in the prompt (should be a bug that already exists now if thats the expectation)

Ah okay, we currently don't remove impossible options from the prompt.

Reply:
Hmm how to handle gitignored files is a good question.
I guess if they're big, we might still want to add them to dvc.
But it's a separate issue.

OK, I think I understand now - if it's ignored by dvc then we don't need to do anything with it, it will just be git added by default.